src/metabase/metabot/settings.clj

```diff
@@ -38,7 +38,9 @@
 (defsetting is-metabot-enabled
   (deferred-tru "Is Metabot enabled?")
   :type :boolean
-  :visibility :public
+  :visibility :internal
```

Comment:
Does this mean it won't be returned to users in the html template? Is it possible to still keep it public because FE needs to understand if Metabot should be rendered

Reply:
Oh sure